src/main/java/org/openstreetmap/atlas/geography/MultiPolygon.java

```diff
@@ -238,29 +238,10 @@ public boolean fullyGeometricallyEncloses(final Location location)
         return false;
     }
 
-    /**
-     * @param polyLine
-     *            A {@link PolyLine} item
-     * @return True if the {@link MultiPolygon} contains the provided {@link PolyLine}.
-     */
-    @Override
-    public boolean fullyGeometricallyEncloses(final PolyLine polyLine)
```

Comment:
Ignore the formatting here, I've double checked my Eclipse settings and ran spotlessApply before pushing the changes. The only worthwhile changes here are lines 336-343 and line 142